files/common/usr/bin/unpack-image

```diff
@@ -130,7 +130,7 @@ rm payload.tar.gz || die "failed to remove payload.tar.gz"
 
 if $opt_x; then
 	sed -i \
-		"s/^\(MINIMUM_REBOOT_OPTIONAL_VERSION\)=.*$/\1=$VERSION/" \
+		"s/^\\(MINIMUM_REBOOT_OPTIONAL_VERSION\\)=.*$/\\1=$VERSION/" \
```

Comment:
I don't see it mentioned in the PR description that you verified this change, so I went ahead and did that..
```
$ sudo unpack-image -x internal-dev.upgrade.tar.gz
Progress increment: 16:33:13:520674317+0000, 10, Extracting upgrade image.
Progress increment: 16:36:17:184840701+0000, 40, Verifying format.
Progress increment: 16:41:01:256204531+0000, 100, Unpacking successful.

$ grep VERSION= /var/dlpx-update/latest/version.info 
VERSION=2019.6.27.23
MINIMUM_VERSION=5.3.0.0
MINIMUM_REBOOT_OPTIONAL_VERSION=2019.6.27.23
```